app/src/main/kotlin/org/fossify/musicplayer/activities/TracksActivity.kt

```diff
@@ -348,7 +348,7 @@ class TracksActivity : SimpleMusicActivity() {
 
     private fun onSearchQueryChanged(text: String) {
         val filtered = tracksIgnoringSearch.filter {
-            it.title.contains(text, true) || ("${it.artist} - ${it.album}").contains(text, true)
+            it.title.normalizeString().contains(text, true) || ("${it.artist} - ${it.album}").normalizeString().contains(text, true)
```

Comment:
Put it on two lines:


```suggestion
            it.title.normalizeString().contains(text, true)
                    || ("${it.artist} - ${it.album}").normalizeString().contains(text, true)
```